sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HivePartitionFilteringSuite.scala

```diff
@@ -418,6 +418,52 @@ class HivePartitionFilteringSuite(version: String)
       dateStrValue)
   }
 
+  test("getPartitionsByFilter: not in chunk") {
```

Comment:
This is the effective test that can catch correctness bugs. Does it test null?

Reply:
The coverage of previous test is not good. I rewrite the test and include you mentioned.